src/main/java/org/broadinstitute/hellbender/tools/exome/CnvCaller.java

```diff
@@ -27,6 +28,8 @@
  *
  * Note:  Assumes that the incoming target file is in log_2(CR)
  * Note:  Assumes that the incoming segment means are CR.
+ * Note: Assumes that the input @{link ReadCountCollection} contains only one sample -- calls will be made for only one
```

Comment:
You have a point. . . I'm not really sure what my opinion is now.  I wouldn't mind if others provided one for me!